Code/Python/readme.md

```diff
@@ -1,3 +1,4 @@
- # Codes in the Python language
+# Codes in the Python language
 
- Tower of Hanoi ---> [Python] (Code/Python/Tower_of_Hanoi.py)
```

Comment:
Remove the space between [Python] add the parenthesis to make it work(same goes for the other line)

Reply:
Done

Reply:
Wait..why have you make changes in this readme file of python/readme....you have make this changes in Algo-Tree/Array/readme.md

Reply:
Oh, sorry I was biased by one of the code linking present in this readme related to Tower of Hanoi code.

Reply:
@Rohan27s The tower of hanoi linking is also incorrect, so I have removed that from the readme, I hope thats fine